js/faq.js

```diff
@@ -1,4 +1,4 @@
-const faqButtons = document.querySelectorAll('.faq__btn');
+const faqButtons = document.querySelectorAll('.faq__btn,.question');
```

Comment:
Since the question also is now selected, wouldn't we want the variable name to be changed correspondingly?
Earlier we were selecting only the buttons and hence the variable name faqButtons was suitable, but that's not the case anymore.

More here: https://discord.com/channels/673083527624916993/736318385201217637/772447122313707540